### ISSUE TITLE
Benzodiazepine dependence

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6534]
**Preferred term label:** Benzodiazepine dependence

**Synonyms**
Benzodiazepine addiction 

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Addictive behavior HP:0030858

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added
